### ISSUE TITLE
The hyper-parameters provides in paper can not reproduce the results

### ISSUE THREAD

**Issue by zanghyu** [manantomar/Mirror-Descent-Policy-Optimization#1]
According to the paper, from table 4, I think the Bregman stepsize is the `klcoeff` in the code, and from table5 we can set tsallis_coeff for each task. As discussed in the paper, the entropy-coeff set 0.2 in all tasks. However, since I run `python run_mujoco.py --env HalfCheetah-v3 --num_timesteps 3e6 --sgd_steps 5 --klcoeff 0.3 --lam 0.2 --tsallis_coeff 0.5 --run 0`, I got negative mean reward at last timesteps. And some other tasks are the same. Do I miss something?

**Comment by manantomar** [comment]
Hi, thanks for the question! A couple of things to note:

1. The code is actually slightly ahead of the current arxiv version, in that the `tsallis_coeff` working range is anything in between 1.0 and 2.0. This was done so as to match the tsallis meaning with other similar papers in literature. The arxiv version will be updated soon with this change.

2. The `sgd_steps` parameter is fixed to 1000 for the off-policy case. This is because we are instantiating off-policy MDPO slightly differently to the multi-step on-policy version, due to computational slowness (see section 5.1 in the paper). Fixing these two should improve your results.

3. A minor comment is on the gym env. We actually test everything on the `-v2` versions. I am not sure how much difference in results this would create.

**Comment by zanghyu** [comment]
Hi, 
Thanks for the reply! I will revise my script and train models again.
